server/cli_list_entrypoint.py

```diff
@@ -5,8 +5,6 @@
 import subprocess
 import textwrap as _textwrap
 
-from girder import logger
-
```

Comment:
We can actually do better than just removing this:
```
try:
   from girder import logger
except ImportError:
   import logging as logger
```
and then leave the log statements in place.